templates/ci-artifacts.template.yml

```diff
@@ -27,8 +27,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: Package project
         run: |
-          cabal v2-sdist --builddir=sdist Cabal cabal-install
-          python3.8 release.py --tarlib=sdist/sdist/Cabal-*.tar.gz --tarexe=sdist/sdist/cabal-install-*.tar.gz
+          cabal v2-sdist --builddir=sdist Cabal cabal-install-solver cabal-install
+          python3.8 release.py --tarlib=sdist/sdist/Cabal-*.tar.gz --tarsolver=sdist/sdist/cabal-install-solver-3.*.tar.gz --tarexe=sdist/sdist/cabal-install-3.*.tar.gz
```

Comment:
why the "3."s?

Reply:
because now that `cabal-install-solver` exists, `cabal-install-*` matches two possible conditions. This allows us to match  on file names more precisely.